codec_specs.md

```diff
@@ -642,6 +644,14 @@ Description: DTS Lossless audio that does not have a core substream. The private
 
 Initialization: none
 
+### A_EAC3
+
+Codec ID: A_EAC3
+
+Codec Name: Dolby Digital Enhanced AC-3
+
+Description: Also known as Dolby Digital Plus. This corresponds to a bitstream with the value of the "Bit Stream Identification" `bsid` in the "Bit Stream Information" structure set to 16 as defined in Annex E Section 2.1 of [@!ATSC-A52] or Section E.1.3.1.6 of [@!ETSI-TS102-366].
```

Comment:
I did not apply this because the EAC3 specs clearly says the bsid should be 16. 11 to 15 seem to be grey area. Also the bsid definition in the Annex doesn't match the definition of the AC3 one (but the bsid position is the same for backward compatibility). So it's better to reference the annex.

Reply:
> 11 to 15 seem to be grey area. 

It is not.
It is exactly as 0 to 7: planned for forward compatibility.

It is not coherent, you accept 0-7 and not 11-15 but htey are defined the same way (provision for compatible streams).

> I did not apply this because the EAC3 specs clearly says the bsid should be 16

It is said that the spec is with 16 only for files adhering to the current spec, with provision that a decoder conform to this spec MUST decoder 11-15.
And FFmpeg, VLC... Do that.

Reply:
Ah yes, I missed that part:

> The bsid field has a value of ‘10000’ (=16) for bitstreams compliant with this Annex. Values of
bsid smaller than 16 and greater than 10 are used for versions of E-AC-3 which are backwards
compatible with version 16 decoders.

Reply:
I listed the values because it's not clear with a "between" if the boundaries are included.